Headers/state_diag_mod.F90

```diff
@@ -113,6 +113,15 @@ MODULE State_Diag_Mod
      TYPE(DgnMap),       POINTER :: Map_SpeciesConc
      LOGICAL                     :: Archive_SpeciesConc
 
+     !%%%%%  ML diagnostics %%%%%
+     REAL(f8),           POINTER :: ConcBeforeChem(:,:,:,:)
```

Comment:
Would you like all of these diagnostics added to the standard model?

Reply:
Yes, I think this diagnostics could be a useful feature for other users and won't create any overhead if not being requested.

Reply:
Sounds good.